doc/src/manual/metaprogramming.md

```diff
@@ -263,29 +263,28 @@ julia> eval(ans)
 julia> ex = :(a + b)
 :(a + b)
 
-julia> eval(ex)
+julia> eval(current_module(), ex)
```

Comment:
should we add some text explaining what `current_module()` is and why it's here?